cs/di-extensions.texy

```diff
@@ -67,6 +67,15 @@ $builder->addDefinition($this->prefix('articles'))
 	->addSetup('setLogger', ['@logger']);
 \--
 
+Pokud potřebujeme kvůli zachování zpětné kompatibilitě přejmenovat službu, můžeme pro ni vytvořit alias. Podobně
+je tomu tak např. v Nette u služby `routing.router`, která je dostupná i pod klíčem `router`.
```

Comment:
Pokud potřebujeme přejmenovat službu, můžeme kvůli zachování zpětné kompatibility vytvořit alias s původním názvem. Podobně
to dělá Nette např. u služby `routing.router`, která je dostupná i pod dřívějším názvem `router`.